rpc/simulation.go

```diff
@@ -29,7 +29,7 @@ func (s *SimulationFlag) UnmarshalJSON(bytes []byte) (err error) {
 
 type SimulatedTransaction struct {
 	TransactionTrace json.RawMessage `json:"transaction_trace,omitempty"`
-	FeeEstimate      FeeEstimate     `json:"fee_estimate,omitempty"`
+	FeeEstimate      FeeEstimate     `json:"fee_estimation,omitempty"`
```

Comment:
Thanks for the fix 👍 don't you mind to rename `FeeEstmate` field into `FeeEstimation` also? no need to rename `FeeEstimate` type, just name of the field.